chart/kubeapps/templates/kubeapps-frontend-config.yaml

```diff
@@ -29,28 +47,23 @@ data:
         return 200 "healthy\n";
       }
 
+      # The default cluster running on the same cluster as Kubeapps.
+      location ~* /api/clusters/default {
+        rewrite /api/clusters/default/(.*) /$1 break;
+        rewrite /api/clusters/default / break;
+        proxy_pass https://kubernetes.default;
+        include "./server_blocks/k8s-api-proxy.conf";
```

Comment:
if `vhost.conf` and `k8s-api-proxy.conf` are at the same level shouldn't this be ` include "./k8s-api-proxy.conf";`?

Reply:
No - I thought so too initially, but they're not relative to the file, they're relative to the CWD, which is set on the CLI or in the bitnami nginx case, configured to `/opt/bitnami/nginx/conf`.